packages/overture-schema-base-theme/tests/bathymetry_baseline_schema.json

```diff
@@ -111,23 +111,19 @@
               "items": {
```

Comment:
I think we should drop these baseline files - they noise up the diffs a lot and the lack of order-preserving-ness means that a small change in the Python can be a big change in the baseline.

We should just trust our tests and process.

If we want to, we can (and probably should) generate JSON Schema as build artifacts (which can be attached to releases).